charmander/stores/pizza_store.py

```diff
@@ -12,29 +12,12 @@
 class PizzaStore(Store):
 
     def __init__(self, graph):
-        super().__init__(self, Pizza)
-
-    def _filter(
-        self,
-        query,
-        customer_id=None,
-        crust_type=None,
-        size=None,
-        **kwargs,
-    ):
-        if customer_id is not None:
-            query = query.filter(
-                Pizza.customer_id == customer_id,
-            )
-
-        if crust_type is not None:
-            query = query.filter(
-                Pizza.crust_type == crust_type,
-            )
-
-        if size is not None:
-            query = query.filter(
-                Pizza.size == size,
-            )
-
-        return super()._filter(query, **kwargs)
+        super().__init__(
+            graph,
+            Pizza,
+            auto_filter_fields=(
```

Comment:
In my first draft of this code, I forgot about a really cool `microcosm-postgres` feature called `auto_filter_fields` which allows the store to automatically support the cookie-cutter filter logic which suffices 90% of the time.